jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/resource/config/template/JpaGroupAppConfigTemplate.java

```diff
@@ -49,7 +49,7 @@
 public class JpaGroupAppConfigTemplate extends ConfigTemplate {
     public static final String GET_GROUP_APP_TEMPLATE_RESOURCE_NAMES = "getGroupAppTemplateResourceNames";
     public static final String GET_GROUP_APP_TEMPLATE_CONTENT = "getGroupAppTemplateContent";
-    public static final String GET_GROUP_APP_TEMPLATE_META_DATA = "getGroupAppTemplateMetaData";
+    public static final String GET_GROUP_APP_TEMPLATE_META_DATA_WITH_APP = "getGroupAppTemplateMetaDataWithApp";
```

Comment:
Please change the name, since the name tells me that you are retrieving the meta data WITH the application. I think what you meant was just get the meta data. GET_GROUP_APP_TEMPLATE_META_DATA is fine since you deleted the old one. Parameters should be seldom used in the name.